Add Rock Band en Español

carlmylo opened the pull request:
Added icon for Rock Band en Español. They've been under this banner since 2011 but only recently started using the origin locale 'rbee' in CON DTAs.

I kept the Spanish name in the English locale because the group prefers it that way. I can change it if needed and use es-US instead.

carlmylo committed:
Add Rock Band en Español

Added icon for Rock Band en Español. They've been under this banner since 2011 but only recently started using the origin locale 'rbee' in CON DTAs.

I kept the Spanish name in the English locale because the group prefers it that way. I can change it if needed and use es-US instead.

kaduwaengertner reviewed (changes requested):
Just some small tweaks on the icon:
![image](https://github.com/user-attachments/assets/6d37e419-160c-4659-970f-719dcfbb6c26)
the white letters have some outline on it, filling with white should solve it, also the blue border doesn't seem to have the effect from the template

carlmylo committed:
Apply suggestions

I do not have the source files so I tried to recreate everything using what was sent.

wyrdough committed:
Merge branch 'master' into rockbandenespañol

wyrdough committed:
Fix formatting, missing comma (using the github editor is a bad idea)
